content/case_histories/bookpurnong/properties.rst

```diff
@@ -3,3 +3,25 @@
 Properties
 ==========
 
+The sandy topsoil at Bookpurnong is mostly saturated by water from the river, irrigation, and other source of recharge. The ground conductivity is therefore dominated by the salinity of groundwater. High salinity in water means more ions acting as free charge carriers, so high electrical conductivity is expected at the place suffers from high salinization (see more information under Physical Properties > Electrical Conductivity). At Bookpurnong, the fresh water is from the river and irrigation, and the saline water is from the natural acquifer under the highland. Therefore, the conductivity can be a direct indicator of the source of groundwater, from the river or the intrusion. 
```

Comment:
" high electrical conductivity is expected at the place **suffering** from high salinization"